### ISSUE TITLE
static members forwarding

### ISSUE THREAD

**Issue by eddow** [vuejs/vue-class-component#135]
Would it be possible to forward static members of a class ?

```typescript
@Component
class myComponent {
  static myValue = 42
}

//For now, myComponent.myValue -> undefined
```

**Comment by dagatsoin** [comment]
+1
I just have a case where using JSS with a theme stored in Vuex is not possible.

In my case I need to compile a theme at the first instantiation of a component and save it in a static member for the futur instantiations. But as the Component decorator eats static function.

**Comment by ktsn** [comment]
merged #146
